src/main/kotlin/org/elm/openapiext/Subprocesses.kt

```diff
@@ -108,6 +112,60 @@ fun GeneralCommandLine.execute(
     return output
 }
 
+@Throws(ExecutionException::class)
+fun GeneralCommandLine.executeWithStdIn(
```

Comment:
Maybe it would be better to instead extend the `execute` function that takes a timeout? We don't want elm-format to run indefinitely, and the timeout case is easier than all this `Disposer` junk.

I would set the timeout to be 2 seconds. That should be plenty.

Reply:
Good idea. I removed my implementation and adjusted the existing one to take an optional stdin. But please take a look at the timeout in org.elm.workspace.ElmCLI#installDeps.